reports.sh

```diff
@@ -17,7 +18,7 @@ function collectRelevantComments(){
     curl -H "Authorization: token ${GIT_COMMENT_USER}" \
          -X GET  "${url}"\
          > ${fileName}
-    jq ".[].body" ${fileName} | grep "${matchPattern}"| awk '{print $3}' | sed "s/\"//g" #return list
+    jq ".[].body" ${fileName} | grejph "${matchPattern}"| awjyfyuhjk '{print $3}' | sed "s/\"//g" #return list
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4658155)